src/lib/ngsi/ContextElementVector.cpp

```diff
@@ -116,15 +116,21 @@ void ContextElementVector::release(void)
 
 /* ****************************************************************************
 *
-* ContextElementVector::get -
+* ContextElementVector::operator -
 */
-ContextElement* ContextElementVector::get(int ix)
+ContextElement* ContextElementVector::operator[](unsigned int ix)
 {
-  return vec[ix];
+    if (ix < vec.size())
+    {
+      return vec[ix];
+    }
+    else
+    {
+      return NULL;
+    }
```

Comment:
unnecessary lines:

    else
     {
         return NULL;
     }

It's enough with:

    return NULL;